tests/test_samples.py

```diff
@@ -499,27 +503,28 @@ def test_merging():
 def test_beta():
     pc = NestedSamples(root="./tests/example_data/pc")
     weight = pc.weight
-    assert_array_equal(pc['weight'], pc.weight)
+    assert_array_equal(weight, pc.weight)
+    assert_array_equal(pc.index.get_level_values(1), pc.weight)
     assert pc.beta == 1
 
     prior = pc.set_beta(0)
     assert prior.beta == 0
-    assert_array_equal(prior['weight'], prior.weight)
+    assert_array_equal(prior.index.get_level_values(1), prior.weight)
```

Comment:
Would using `'weight'` instead of `1` be more future proof, here and in the following lines?